src/screens/signUp.js

```diff
@@ -5,6 +5,10 @@ import { firebase } from '../firebase/config';
 
 import Loading from '../components/loading';
 
+// ! DEBUG FUNCTIONS ROUTE
```

Comment:
What are this comments?

Reply:
In this case, that import is only to help me debug (the function log message and states if needed with timestamp so i see better when everything its happening), so the import and all instances of the function should be deleted for the final version and the comment its to help remember that

In the future i'll change it to // ! DELETE AFTER DEBUGGING SESSION